lib/AST/ASTPrinter.cpp

```diff
@@ -2768,7 +2768,7 @@ void PrintAST::visitFuncDecl(FuncDecl *decl) {
     if (!Options.SkipIntroducerKeywords) {
       if (decl->isStatic() && Options.PrintStaticKeyword)
         printStaticKeyword(decl->getCorrectStaticSpelling());
-      if (decl->isMutating() && !decl->getAttrs().hasAttribute<MutatingAttr>()) {
```

Comment:
Not anymore I believe, since I chose to skip mutability modifiers in `printAttributes`.

Reply:
FYI this caused interface generation to print `mutating mutating` for explicitly mutating functions, because the changes made to `printAttributes` only applied to *implicit* attributes.  Fixed: https://github.com/apple/swift/pull/30183